app/controllers/metrics_controller.rb

```diff
@@ -273,4 +285,11 @@ def new_metrics_params
                                      grade_threshold count_threshold
                                    ])
   end
+
+  def permission_check
+    return if @cud.course_assistant && @course.watchlist_configuration.allow_ca || @cud.instructor
```

Comment:
Same comment about the safe navigation operator (see the comment on `assessments/index.html.erb`)
![Screenshot 2022-11-25 at 02 15 06](https://user-images.githubusercontent.com/9074856/203922890-31ff75bc-c84f-495f-b865-2491b058effa.png)